server/executor/pollingprofile/polling_profile_resource.go

```diff
@@ -40,13 +50,31 @@ type PeriodicPollingConfig struct {
 	Timeout    string `mapstructure:"timeout"`
 }
 
+func (ppc *PeriodicPollingConfig) Validate() error {
+	if ppc == nil {
+		return fmt.Errorf("missing periodic polling profile configuration")
+	}
+
+	if _, err := time.ParseDuration(ppc.RetryDelay); err != nil {
+		return fmt.Errorf("retry delay configuration is invalid: %w", err)
+	}
+
+	if _, err := time.ParseDuration(ppc.Timeout); err != nil {
+		return fmt.Errorf("timeout configuration is invalid: %w", err)
+	}
+
+	return nil
+}
```

Comment:
Just noticed we weren't validating if the strings were actually durations.

Reply:
Question: do we intend to allow users to write durations more relaxedly as we do on assertions, like "1 d" instead of "1d"?